nimble/host/src/ble_gap.c

```diff
@@ -3584,7 +3584,7 @@ ble_gap_ext_connect(uint8_t own_addr_type, const ble_addr_t *peer_addr,
     }
 
     if (duration_ms != BLE_HS_FOREVER) {
-        rc = os_time_ms_to_ticks(duration_ms, &duration_ticks);
+        rc = ble_npl_time_ms_to_ticks(duration_ms, &duration_ticks);
```

Comment:
I fixed this together with multiple other problems on ported code: https://github.com/apache/mynewt-nimble/pull/113

Reply:
good